permissions/plugin-redis-notifier.yml

```diff
@@ -5,3 +5,4 @@ paths:
 - "io/jenkins/plugins/redis-notifier"
 developers:
 - "d3camp0s"
+- "dkmpos89"
```

Comment:
```Exception in thread "main" java.lang.IllegalStateException: User name not known to Artifactory: dkmpos89```

The person needs to login to [Artifactory](https://repo.jenkins-ci.org/webapp/#/login)